src/main/java/solvd/laba/ermakovich/hu/event/CreateDoctor.java

```diff
@@ -9,12 +9,12 @@
  * @author Ermakovich Kseniya
  */
 @Data
-public class CreateDoctorEvent extends Event {
+public class CreateDoctor extends Event {
 
     public static final String EVENT_TYPE = "CreateDoctor";
 
     @SneakyThrows
-    public CreateDoctorEvent(String aggregateId, Doctor doctor) {
+    public CreateDoctor(String aggregateId, Doctor doctor) {
         super(EVENT_TYPE, aggregateId);
         this.setPayload(new ObjectMapper().writeValueAsString(doctor));
```

Comment:
Your ctors should be code-free. Read this https://www.yegor256.com/2015/05/07/ctors-must-be-code-free.html

Reply:
I've created secondary ctors